app/Http/Controllers/UserController.php

```diff
@@ -54,11 +54,11 @@ public function updatePassword(Request $request)
     {
         $user = Auth::user();
 
-        $validator = Validator::make($request->all(), [
+        $validator = Validator::make($request->except('_token'), [
             'old_password' => 'required|string|password',
             'new_password' => 'required|string|min:8|confirmed|different:old_password',
         ]);
-
+        $validator->validate();
```

Comment:
Actually, I just forgot to remove this part, sorry. You can keep either that or the one below.